src/HttpClient.ts

```diff
@@ -153,9 +166,16 @@ export interface IHttpConfig {
     proxy?: IProxyConfig | false;
     headers?: { [key: string]: string };
     retries?: number;
+    retryDelay?: number;
     timeout?: number;
+    retryOnStatusCode?: RetryOnStatusCode;
 }
 
+export const retryOnStatusCodeSso: (statusCode: number) => boolean = (statusCode: number): boolean => {
+    return statusCode === 400;
+};
+
+export type RetryOnStatusCode = typeof retryOnStatusCodeSso;
```

Comment:
```suggestion
```

Reply:
In the future we will have move types that will be added here